@types/node Class 'Module' incorrectly implements interface 'NodeModule'.

- [x] I tried using the `@types/node` package and had problems.
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @devoto13, @andy-ms 

Compilation cannot succeed with the version 8.0.27 installed. The error is
`node_modules/@types/node/index.d.ts(5615,11): error TS2420: Class 'Module' incorrectly implements interface 'NodeModule'. Property 'hot' is missing in type 'Module'.`

Version 8.0.26 does not have this error.

--- a/types/webpack-env/index.d.ts
+++ b/types/webpack-env/index.d.ts
@@ -1,6 +1,7 @@
 // Type definitions for webpack (module API) 1.13
 // Project: https://github.com/webpack/webpack
 // Definitions by: use-strict <https://github.com/use-strict>
+//                 rhonsby <https://github.com/rhonsby>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /**
@@ -60,7 +61,7 @@ declare namespace __WebpackModuleApi {
         loaded: boolean;
         parent: any;
         children: any[];
-        hot: Hot;
+        hot?: Hot;
     }
     type ModuleId = string|number;
 

